opm/autodiff/WellStateFullyImplicitBlackoil.hpp

```diff
@@ -57,7 +59,7 @@ namespace Opm
         void init(const Wells* wells, const State& state, const PrevState& prevState)
         {
             // clear old name mapping
-            wellMap_.clear();
+            wellMap().clear();
```

Comment:
It is called on line 73 below, so I assume you refer to the early returns for the no-wells case(s)? I think a better solution is to move the BaseType::init() call to the top of this method. That also makes it a little more obvious that we are first calling the base implementation and then adding stuff.